docs-gen/hugo.toml

```diff
@@ -38,6 +33,15 @@ canonifyURLs = true
   github_subdir = "docs-gen"
   github_branch = "master"
   github_project_repo = "https://github.com/sky-uk/kfp-operator"
+  subpath = "/kfp-operator"
```

Comment:
can we use this variable in `github_project_repo` and `baseURL`?

Reply:
I can't think of a nice way to do it; TOML doesn't support variables directly